stubs/resources/views/auth/login.blade.php

```diff
@@ -33,11 +42,9 @@
             </div>
 
             <div class="flex items-center justify-end mt-4">
-                @if (Route::has('password.request'))
-                    <a class="underline text-sm text-gray-600 hover:text-gray-900" href="{{ route('password.request') }}">
-                        {{ __('Forgot your password?') }}
-                    </a>
-                @endif
+                <a class="underline text-sm text-gray-600 hover:text-gray-900" href="{{ route('register') }}">
+                    {{ __("Don't have an account yet?") }}
+                </a>
```

Comment:
Hmm, not sure if that's ok with the team. Will leave it to the maintainers since the check wasn't needed till now (although it might me a reasonable change).